docs/src/components/ReactExample/components/Board.tsx

```diff
@@ -96,6 +104,18 @@ const Board = ({
               Playground
             </ButtonLink>
           )}
+          {variants.length > 0 && (
+            <Popover content={<Variants exampleVariants={variants} onChange={onChangeVariant} />}>
+              <ButtonLink
+                type="secondary"
+                asComponent="button"
+                ariaExpanded={isVariantsOpened}
+                iconRight={isVariantsOpened ? <ChevronUp /> : <ChevronDown />}
+              >
+                Variants
+              </ButtonLink>
```

Comment:
Clicking on this button throws off the scrolling position. Probably should have been a Button, not ButtonLink, no?

Reply:
nope, fixed that already, should be a buttonlink visually, but `asComponent="button"`